packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -509,6 +509,35 @@ async function getOriginalRescheduledBooking(uid: string, seatsEventType?: boole
   });
 }
 
+//checks if booking can be rescheduled
+function checkIfRescheduleAllowed(booking: Awaited<ReturnType<typeof getOriginalRescheduledBooking>>) {
+  //check if the booking was found
```

Comment:
It's a good idea to create `checkIfRescheduleAllowed` function. However, it's possible to replace comments like "//check if the booking was found" by creating new functions:
```
checkIfBookingExists()
checkIfBookingEnded()
checkIfBookingCancelled()
```

Then, `checkIfRescheduleAllowed` would invoke all of them and there is no need for adding `//` comments because the function names clearly describe what they do.